Bug 1710828 - Document InvalidType error [doc only] (#1968)
mozilla/glean: docs/user/reference/metrics/timespan.md

#### Recorded errors

> * [`invalid_value`](../../user/metrics/error-reporting.md): if you attempt to record a negative elapsed duration.
> * [`invalid_state`](../../user/metrics/error-reporting.md): if you call this API after calling `start` or call this API multiple times.

Reviewer: we don't use `you` anywhere else, so let's fix it here.

```suggestion
* [`invalid_value`](../../user/metrics/error-reporting.md): if attempting to record a negative elapsed duration.
* [`invalid_state`](../../user/metrics/error-reporting.md): if this method is called after calling `start` or this method is called multiple times.
```

> * [`invalid_type`](../../user/metrics/error-reporting.md): if a negative, floating point or non-number value is given.

Reviewer: as above. negative is `invalid_value`.

Author: ```suggestion
* [`invalid_type`](../../user/metrics/error-reporting.md): if a floating point or non-number value is given.
```
